bothub/common/models.py

```diff
@@ -284,6 +291,12 @@ class Meta:
         max_length=32,
         help_text=_("Easy way to found and share repositories"),
     )
+    type = models.CharField(
+        _("type"),
+        max_length=50,
+        choices=TYPE_CHOICES,
+        default=TYPE_CLASSIFIER,
+    )
```

Comment:
We should a void to use reserved or builtin words to name variables or attributes. Can you rename it to "repository_type"?